Cabal-syntax/src/Distribution/Fields/Pretty.hs

```diff
@@ -59,7 +59,7 @@ showFields rann = showFields' rann (const id) 4
 -- | 'showFields' with user specified indentation.
 showFields'
   :: (ann -> CommentPosition)
-     -- ^ Convert an annotation to lined to preceed the field or section.
+     -- ^ Convert an annotation to a line preceding the field or section.
```

Comment:
This doesn't look like what that function is supposed to do

Reply:
proposing "Convert a comment annotation to a line before or after the field or section"